packages/botonic-plugin-flow-builder/src/webview/use-webview-contents.ts

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+
+import {
+  UseWebviewContents,
+  UseWebviewContentsProps,
+  WebviewContentsResponse,
+  WebviewContentType,
+  WebviewImageContent,
+  WebviewTextContent,
+} from './types'
+
+export function useWebviewContents({
+  flowBuilderApiUrl,
+  version,
```

Comment:
I think we should modify the webpack to have this FLOW_BUILDER_API_URL as an environment variable, this way it could be an optional parameter and only the version would be needed. Do you think this is a good idea?

Reply:
It make sense, to use it the same way Botonic does with the `HUBTYPE_API_URL` and set it to the production one by default if no environment variable is set.